lib/internal/modules/esm/resolve.js

```diff
@@ -690,8 +696,13 @@ function packageImportsResolve(name, base, conditions) {
   const packageConfig = getPackageScopeConfig(base);
   if (packageConfig.exists) {
     packageJSONUrl = pathToFileURL(packageConfig.pjsonPath);
-    const imports = packageConfig.imports;
+    let imports = packageConfig.imports;
```

Comment:
nit: (I see you only changed `const` → `let`, so feel free to throw back an "out of scope") destructuring is generally preferred (it's more concise)

```suggestion
    let { imports } = packageConfig;
```